### PR TITLE
Rewrite Get Started for the Collector and separate install instructions

### PR DESCRIPTION
**Pull request by theletterf**
This solves #1695 and potentially a few others.

I went for the simplest possible approach:

- Download the binary or Docker image
- Run with default configuration
- Install telemetrygen
- Send traces
- Check in zpages

At the same time I've split the install instructions into a separate page.

The problem I'm seeing with using telemetrygen is that the official install instructions don't work (https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27855) and that we don't provide a binary (https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27856). I think the first issue should be solved, at least. 

Note: I'm getting some errors in a blog post that I haven't touched...

### CODE REVIEW

**Comment by theletterf** [comment]
@mx-psi How could I replace the telemetrygen instructions so that they'd work? Getting rid of the Go dependency for the get started would be nice, but I guess that doing it all in docker would require creating a docker network first?

**Comment by mx-psi** [comment]
> @mx-psi How could I replace the telemetrygen instructions so that they'd work? Getting rid of the Go dependency for the get started would be nice, but I guess that doing it all in docker would require creating a docker network first?

I think we should address open-telemetry/opentelemetry-collector-contrib#27855 and fix the `go install`

File: content-modules/community
Comment (svrnm) [inline content-modules/community:1]: make sure that your PR does not contain updates to the content-modules.

File: content/en/docs/collector/getting-started.md [hunk 1]
```diff
@@ -1,358 +1,90 @@
...
+cSpell:ignore: otelcol telemetrygen docker jaeger zipkin prometheus dpkg
 weight: 1
 ---
 
-If you aren't familiar with the deployment models, components, and repositories
-applicable to the OpenTelemetry Collector, first review the [Data Collection][]
-and [Deployment Methods][] page.
+The OpenTelemetry Collector listens to metrics, logs, and traces, processes the
+telemetry, and exports it to a wide variety of observability back-ends using its
+components. For a conceptual overview of the Collector, see [Collector][].
```
Comment (svrnm) [inline content/en/docs/collector/getting-started.md:10]:
```suggestion
components. For a conceptual overview of the Collector, read the [introduction][].
```

File: content/en/docs/collector/getting-started.md [hunk 2]
```diff
@@ -1,358 +1,90 @@
...
 
-## Demo
+The following tutorial shows how to deploy the Collector in agent mode and send
```
Comment (svrnm) [inline content/en/docs/collector/getting-started.md:12]:
```suggestion
The following tutorial shows how to deploy the Collector and send
```

I think at this place the deployment pattern is not relevant and leads to more confusion

File: content/en/docs/collector/getting-started.md [hunk 3]
```diff
@@ -1,358 +1,90 @@
...
+telemetry to it using the default configuration and the telemetrygen utility.
```
Comment (svrnm) [inline content/en/docs/collector/getting-started.md:13]:
```suggestion
telemetry to it using the default configuration and the [telemetrygen](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/telemetrygen) utility.
```

File: content/en/docs/collector/getting-started.md [hunk 4]
```diff
@@ -1,358 +1,90 @@
...
-
-{{< tabpane text=true >}} {{% tab AMD64 %}}
-
-```sh
-sudo apt-get update
-sudo apt-get -y install wget systemctl
-wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_amd64.deb
-sudo dpkg -i otelcol_{{% param collectorVersion %}}_linux_amd64.deb
-```
+To follow this tutorial you need the following Go 1.20 or higher.
```
Comment (svrnm) [inline content/en/docs/collector/getting-started.md:17]:
```suggestion
To follow this tutorial you need the following

*  Go >= 1.20
```

File: content/en/docs/collector/getting-started.md [hunk 5]
```diff
@@ -1,358 +1,90 @@
...
-wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_386.deb
-sudo dpkg -i otelcol_{{% param collectorVersion %}}_linux_386.deb
-```
+   Alternatively, you can download and run the Docker container:
 
-{{% /tab %}} {{< /tabpane >}}
+   ```sh
+   docker pull otel/opentelemetry-collector-contrib:{{% param collectorVersion %}}
+   docker run otel/opentelemetry-collector-contrib:{{% param collectorVersion %}}
+   ```
```
Comment (svrnm) [inline content/en/docs/collector/getting-started.md:36]: Let's keep it simple here and tell people that for the sake of the getting started they should use docker and they will find different deployment patterns on that other page.

Note: let's not use contrib here, I don't think you will make use of any additional feature of it?

Comment (svrnm) [reply content/en/docs/collector/getting-started.md:36]: Also, add some notes on what they should see on their console

File: content/en/docs/collector/getting-started.md [hunk 6]
```diff
@@ -1,358 +1,90 @@
...
-## Windows Packaging
-
-Windows [releases][] are packaged as gzipped tarballs (`.tar.gz`) and will need
-to be unpacked with a tool that supports this compression format.
-
-Every Collector release includes an `otelcol.exe` executable that you can run
-after unpacking.
-
-## Local
+You can find more information about the demo in the [README.md][] file.
```
Comment (svrnm) [inline content/en/docs/collector/getting-started.md:68]: Let's remove that section about the demo

**Review by svrnm — CHANGES_REQUESTED** [review]
Thanks for taking the time to write this down @theletterf! Some change requests and suggestions

File: content/en/docs/collector/getting-started.md (hunk 5 above)
Comment (theletterf) [reply content/en/docs/collector/getting-started.md]: Solved in https://github.com/open-telemetry/opentelemetry.io/pull/3412/commits/5826d97fc3f8b83911360fa7fa850fa91f1576b1

Note that at least two ports must be exposed when running the Docker container

File: content/en/docs/collector/getting-started.md (hunk 6 above)
Comment (theletterf) [reply content/en/docs/collector/getting-started.md]: Done in https://github.com/open-telemetry/opentelemetry.io/pull/3412/commits/c1b457308a14b07867b4f414b2ba5ff67cdbb541

**Comment by theletterf** [comment]
Closing this and opening https://github.com/open-telemetry/opentelemetry.io/pull/3426 due to pesky submodule issues...

**Pull request closed without merging**